docs/features/software-templates/extending/create-your-own-templater.md

```diff
@@ -87,9 +87,10 @@ follows:
 _note_ Currently the templaters that we provide are basically Docker action
 containers that are run on top of the skeleton folder. This keeps dependencies
 to a minimal for running backstage scaffolder, but you don't /have/ to use
```

Comment:
```suggestion
to a minimal for running backstage scaffolder, but you don't _have_ to use
```

I know this wasn't part of your PR, but since you're here :)

![image](https://user-images.githubusercontent.com/33203301/104990052-cc9c9d00-59e9-11eb-8967-f4fd75866a70.png)